plugins/out_pgsql/pgsql_loadlib.h

```diff
@@ -0,0 +1,163 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2020 The Fluent Bit Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef FLB_OUT_PGSQL_LOADLIB_H
+#define FLB_OUT_PGSQL_LOADLIB_H
+
+#ifdef _WIN32
+
+HMODULE pqDll;
+
+typedef enum
```

Comment:
Why these enum are not imported from libpq-fe.h?

Reply:
Talking with a PostgreSQL developers this seems a bit weird and really a problem, since these ENUMs may/will change in the future and this will not be reflected this. Is there a way to handle the "can't be found" issue  like with a warning/error? I'm a Linux user and stop programming stuff for win32 a couple of decades ago, so my knowledge of win32 is a bust rusty

Reply:
Sorry, there was something I haven't mentioned in the previous comment. Wrong libpq.dll version (x86/x64) found on the system is also resulting in an error at start-up. You can see the two behaviors described in this PR's description https://github.com/fluent/fluent-bit/pull/3811.